qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -187,6 +187,9 @@ set_helm_params() {
             HELM_PARAMS+=(--set "kube.external_ips[$i]=${external_ips[$i]}")
         done
     fi
+    if [ "${EMBEDDED_UAA:-false}" = "true" ]; then
```

Comment:
[[]] for consistency

Reply:
Fixed in 86a2d8d13026096aad7fecfcb1d52bb81fe4f16e along with the following `if`s.